src/pages/Board/index.tsx

```diff
@@ -32,30 +48,46 @@ const Board = () => {
             작성하기
           </S.WriterButton>
         </S.WriteContainer>
-        {notifications.map((data, i) => (
-          <S.BoardBoxContainer
-            key={i}
-            onClick={() => {
-              router(`/boards/${i}`);
-            }}
-          >
+        {projectDetail?.map((data, i) => (
+          <S.BoardBoxContainer key={i}>
             <S.ProfileInfoContainer>
-              <div>{data.project}</div>
-              <div>{data.user}</div>
+              <S.Title
+                onClick={() => {
+                  router(`/boards/${i}`);
+                }}
+              >
+                {data.title}
+              </S.Title>
+              <S.UserProfile>
+                <S.ProfileImg
+                  src={data.userImg}
+                  alt='img'
+                  onClick={() => {
+                    router(`/others/${data.userId}`);
+                  }}
+                />
+                {data.userName}
+              </S.UserProfile>
             </S.ProfileInfoContainer>
             <S.ProjectDetail>
-              <S.DetailDay>{data.timestamp}</S.DetailDay>
+              <S.DetailDay>
+                {truncateText(String(data.content), 30)}
+              </S.DetailDay>
             </S.ProjectDetail>
             <S.Detail>
               <S.DetailBox>
                 <EyeIcon color={theme.grey[600]} />
-                10234
+                {Number(data.viewCount) < 1 ? 0 : data.viewCount}
               </S.DetailBox>
               <S.DetailBox>
                 <MessageIcon color={theme.grey[600]} />
                 fds
               </S.DetailBox>
-              <S.DetailDay>2023일 전</S.DetailDay>
+              <S.DetailDay>
+                {daysAgo(data.createTime) < 1
```

Comment:
```suggestion
                {formatDay(data.createTime) < 1
```

daysAgo는 함수의 역할과는 부적절한것같아요 더 좋은 함수명을 찾을수있을것같아용